### ISSUE TITLE
remove 'React' vs 'react' warning

### ISSUE THREAD

**Issue by viviancpy** [maxmantz/redux-oidc#2]
Can I request the redux-oidc/dist/redux-oidc.js file to use 

`define(["react"]` 
instead of 
`define(["React"] `

This is to remove the warnings like:

```
WARNING in ./~/react/lib/getNodeForCharacterOffset.js
There is another module with an equal name when case is ignored.
```

**Comment by maxmantz** [comment]
Sure thing. Will fix it asap.

**Comment by maxmantz** [comment]
It should be fixed in the latest version (1.1.14). Please let me know if this fixed your issue.
